src/libsyntax/feature_gate.rs

```diff
@@ -551,6 +551,8 @@ declare_features! (
     (accepted, match_beginning_vert, "1.25.0", Some(44101), None),
     // Nested groups in `use` (RFC 2128)
     (accepted, use_nested_groups, "1.25.0", Some(44494), None),
+    // Termination trait in main (RFC 1937)
+    (accepted, termination_trait, "1.25.0", Some(43301), None),
```

Comment:
The build for `accepted` is when it was accepted, right?  Which I think will now be 1.26.

Also, you could consider making this feature be `termination_trait_main`, which would let all the remaining checks (and attributes in the minimal nightly code) stay unchanged.

Reply:
I wasn't sure about this. The tracking issue (#48453) has milestone 1.25.

re: `termination_trait_main`, it makes a little more sense since the trait itself is still not stable. However, it makes just as much sense to me to keep the trait for tests as it is now in the PR. Then we'd have `termination_trait_main`, `termination_trait_lib`, and `termination_trait_test`.

Reply:
After re-reading the [Forge guide](https://forge.rust-lang.org/stabilization-guide.html), it should definitely be 1.26. Updated.

For the feature name, I see where you're coming from more now: pull out the thing being stabilized, and leave the unstable stuff in the "main feature." I have no strong preference here at all, and went with the [initial recommendation](https://github.com/rust-lang/rust/issues/48453#issuecomment-372323844). If two people agree on what it should be, or one person feels more strongly and no one objects, I'll change it to that!

Reply:
Good by me!  (I try to say "consider" for things I'm not that concerned about.)